src/components/Card/Card.vue

```diff
@@ -42,29 +64,59 @@ export default {
+        return 'background: linear-gradient(145deg, #23d284, #1eb06f)'
+      } else if (this.aqi >= 51 && this.aqi <= 100) {
+        return 'background: linear-gradient(145deg, #ffdd29, #dbba22)'
+      } else if (this.aqi >= 101 && this.aqi <= 150) {
+        return 'background: linear-gradient(145deg, #ff7e23, #e66a1e)'
+      } else if (this.aqi >= 151 && this.aqi <= 200) {
+        return 'background: linear-gradient(145deg, #e42c88, #c02572)'
+      } else if (this.aqi >= 201 && this.aqi <= 300) {
+        return 'background: linear-gradient(145deg, #7e1ee3, #6a19bf)'
+      } else if (this.aqi >= 301) {
+        return 'background: linear-gradient(145deg, #880225, #720220)'
+      }
     },
-    refreshLocation () {
-      const cityState = this.locale.split(', ')
-      const location = { city: cityState[0], state: cityState[1] }
-      this.$parent.$emit('refreshLocale', location)
+    determineAqiRating () {
+      if (this.aqi <= 50) {
+        return 'Good'
+      } else if (this.aqi >= 51 && this.aqi <= 100) {
+        return 'Moderate'
+      } else if (this.aqi >= 101 && this.aqi <= 150) {
+        return 'Unhealthy for Sensitive Groups'
+      } else if (this.aqi >= 151 && this.aqi <= 200) {
+        return 'Unhealthy'
+      } else if (this.aqi >= 201 && this.aqi <= 300) {
+        return 'Very Unhealthy'
+      } else if (this.aqi >= 301) {
+        return 'Hazardous'
+      }
     },
-    determineAqiMessage () {
+    determineAqiText () {
       if (this.aqi <= 50) {
-        return '🟢 Good'
+        return 'It\'s a great day to be active outside!'
       } else if (this.aqi >= 51 && this.aqi <= 100) {
-        return '🟡 Moderate'
+        return 'It\'s a good day to be active outside! People who are unusually sensitive to air pollution could have symptoms.'
       } else if (this.aqi >= 101 && this.aqi <= 150) {
-        return '🟠 Unhealthy for Sensitive Groups'
+        return 'It\'s OK to be active outside for short durations. For longer activities, take more breaks and lower the intensity. Members of sensitive groups may experience health effects. Watch for symptoms and take action as needed.'
       } else if (this.aqi >= 151 && this.aqi <= 200) {
-        return '🔴 Unhealthy'
+        return 'For all outdoor activities, take more breaks and lower the intensity. Consider moving longer or more intense activities indoors or rescheduling them to another day or time. Some members of the general public may experience health effects; members of sensitive groups may experience more serious health effects.'
       } else if (this.aqi >= 201 && this.aqi <= 300) {
-        return '🟣 Very Unhealthy'
+        return 'Health alert: The risk of health effects is increased for everyone. Move all activities indoors or reschedule them to another day.'
       } else if (this.aqi >= 301) {
-        return '💀 Hazardous'
+        return 'Health warning of emergency conditions: everyone is more likely to be affected.'
       }
+    },
```

Comment:
These messages look great, and I think should really help set the intentions of our project apart from others utilizing the same air quality API